app/design/frontend/Elogic/maksym/Magento_Catalog/layout/default.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+
+        <referenceContainer name="sidebar.additional">
+
+            <block class="Magento\Cms\Block\Block" name="promotion-block" after="sidebar.main">
+                <arguments>
+                    <argument name="block_id" xsi:type="string">promotion-block</argument>
+                </arguments>
+            </block>
+
+        </referenceContainer>
+
+    </body>
+</page>
```

Comment:
`promotion-block` should be added only to catalog pages, use another xml file for this instruction